test/built-ins/Promise/any/ctx-non-ctor.js

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 2019 Sergey Rubanov. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: >
+  Promise.any invoked on a non-constructor value
+esid: sec-promise.any
+info: |
+  ...
+  2. Let promiseCapability be ? NewPromiseCapability(C).
+
+  NewPromiseCapability ( C )
+
+  1. If IsConstructor(C) is false, throw a TypeError exception.
+
+features: [Promise.any]
```

Comment:
Really? Symbol is 5 years old, what engines don't support Symbol and need to filter against the feature?

Reply:
Why have an incomplete feature list? It seems much better to forever use feature tags than to arbitrarily omit information on a moving time window, based on whenever a PR happens to land.

Reply:
A few years ago everyone complained that we were too strict about the inclusion of feature flags ("It's too hard to contribute" lol), so we started trying to relax the expectation by saying "If the feature is > 2 years old, then we don't require inclusion in _new_ tests". 

One day I'd love to finish my project that automates the inclusion of feature tags in front matter 🤷‍♂

Reply:
Hopefully that automation can back-add all the missing feature tags.

Reply:
Somehow I want to assume if you have `Promise.any` implemented you also need to have features released in 2+ years. If we want to allow conformant implementations of ECMAScript subsets we should plan it first on ECMA-262, time invested in Test262 needs to be optimized because it's limited.